### ISSUE TITLE
Natural Gas Engines ignore gas fuel value, burns for 100 Kj/unit

### ISSUE THREAD

**Issue by willcheat** [Dysoch/DyWorld#548]
**Describe the bug**
Natural Gas Engine burns its fuel at a rate of 100 Kj/unit, regardless of the energy value of its fuel

Example : 
-Feed a Natural Gas Engine with 100 units (1 pipe worth) of natural gas (50 Mj/unit) should give an expected energy value of 5 000 Mj
-Add 1 basic radar (300 Kj/s energy drain) to said Natural Gas Engine. Expected gas consumption is 1 unit every 166 1/3 seconds (50 Mj / 300 Kj). Real usage is 3 unit / second

**Information (Mods active):**
Basically all needed mods + even distribution 1.0.10
![image](https://user-images.githubusercontent.com/15983362/185298507-2bf5dbeb-de2d-49eb-8c33-a1253f8ba6ca.png)

**Additional context**
100 Kj/unit also applies to gasoline and petroleum. Have not tested any other gases, but I suspect this applies to any other burnable gas
